aws-sdk-v2/src/test/java/io/micronaut/aws/sdk/v2/RekognitionConfigurationProperties.java

```diff
@@ -0,0 +1,24 @@
+package io.micronaut.aws.sdk.v2;
+
+
+import io.micronaut.context.annotation.ConfigurationBuilder;
+import io.micronaut.context.annotation.ConfigurationProperties;
+import software.amazon.awssdk.services.rekognition.RekognitionClient;
+import software.amazon.awssdk.services.rekognition.RekognitionClientBuilder;
+import software.amazon.awssdk.services.s3.S3Configuration;
+
+//tag::class[]
+@ConfigurationProperties(RekognitionClient.SERVICE_NAME)
+public class RekognitionConfigurationProperties  {
```

Comment:
Most ConfigurationProperties classes I made are empty since the `@EachProperty` change, I did include what was done in the S3ConfigurationProperties. Not sure if every single service needs one of these would have Builders

Reply:
So if they leverage underlying SDK builders, then I think it is justified to have them. Otherwise it's pointless to keep them, IMO

Reply:
Sounds good that’s what I was thinking too